.github/workflows/core.yml

```
@@ -77,4 +76,18 @@ jobs:
           install-target: .[dev]
       - name: Run pytest
         run: |
-          pytest -v
+          pwd
+          ls idaes_examples
+          pytest -v idaes_examples
+      - name: Upload pytest report log
+        if: success() || failure()
```

```suggestion
        if: always()
```

https://docs.github.com/en/actions/learn-github-actions/expressions#always

---

Not exactly the same though. I don't want to bother uploading artifacts if the user aborted the job.